components/inputs/Captcha.tsx

```diff
@@ -0,0 +1,59 @@
+import WebView from 'react-native-webview'
+
+import { environment } from '@/environment'
+
+type Props = {
+  onSuccess: (token: string) => void
+  onFail: () => void
+}
+
+const Captcha = ({ onSuccess, onFail }: Props) => {
+  const handleMessage = (event: { nativeEvent: { data: string } }) => {
```

Comment:
Is there a reason not to use `WebViewNativeEvent` from `react-native-webview` typings?